File: Java/OOP/Polymorphism/Method overriding or Runtime polymorphism.java
class Parent {
    void address() {
        System.out.println("I'm from Dhaka, Bangladesh");
    }
}
class Child extends Parent {
    void address() {
        System.out.println("I'm from Chittagong, Bangladesh");
    }
}
class runTime {
    public static void main(String[] args){
        Parent Pr = new Parent();
        Pr.address();
        
        Child Ch = new Child();
        Ch.address();
    }
}

File: C++/OOP/Polymorphism/Method overriding or Runtime polymorphism.cpp
#include <iostream>
using namespace std;

class Parent {
    public:
        void address() {
            cout << "I'm from Dhaka, Bangladesh" << endl;
        }
};
class Child : public Parent {
    public:
        void address() {
            cout << "I'm from Chittagong, Bangladesh" << endl;
        }
};

int main(){
    Parent Pr;
    Pr.address();

    Child Ch;
    Ch.address();
}